hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,61 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ "$platform" == "windows/amd64" ]]; then
+        os::build::version::get_vars
```

Comment:
Yep, sounds good. If I have the bandwidth I'll also dig a little bit about this today. Tks @shiywang.